front/src/components/boxs/weather/WeatherBox.jsx

```diff
@@ -287,7 +287,7 @@ class WeatherBoxComponent extends Component {
       <WeatherBox
         {...props}
         weather={weather}
-        weather_icon={weather_icon}
+        weatherIcon={weatherIcon}
```

Comment:
This breaks the weather icon on my side. Variable should be modified in the `WeatherBox` component too

Reply:
Fixed